src/index.ts

```diff
@@ -92,9 +92,21 @@ export function apply(ctx: Context, config: Config) {
 
       try {
         // send a message and wait for the response
-        const { conversationId, messageId } = conversations.get(key) ?? {}
-        const response = await api.sendMessage({ message: input, conversationId, messageId })
-        conversations.set(key, { conversationId: response.conversationId, messageId: response.messageId })
+        const conversation: Conversation = conversations.get(key) ?? { message: '', conversationId: undefined, messageId: undefined }
```

Comment:
Keep it `undefined`, and then check it below.
Don't mess up the typing with `??`

Reply:
like this?
```ts
let conversation: Conversation = conversations.get(key)
if (!conversation) conversation = { message: '', conversationId: undefined, messageId: undefined }
```